packages/rspack/src/builtin-plugin/index.ts

```diff
@@ -16,12 +16,21 @@ export * from "./ArrayPushCallbackChunkFormatPlugin";
 export * from "./CommonJsChunkFormatPlugin";
 export * from "./ModuleChunkFormatPlugin";
 export * from "./HotModuleReplacementPlugin";
+export * from "./WebWorkerTemplatePlugin";
 
 export * from "./HtmlRspackPlugin";
 export * from "./CopyRspackPlugin";
 export * from "./SwcJsMinimizerPlugin";
 export * from "./SwcCssMinimizerPlugin";
 
+import { LimitChunkCountPlugin } from "./LimitChunkCountPlugin";
+
+export const optimize = {
+	get LimitChunkCountPlugin() {
+		return LimitChunkCountPlugin;
+	}
+};
+
```

Comment:
```suggestion
export * from "./LimitChunkCountPlugin";
```

and move to line 20

Reply:
My code is based on the fact that webpack use plugins in the following way:

```js
var webpack = require("webpack")

new webpack.optimize.LimitChunkCountPlugin({
    maxChunks: 1
})
```

Reply:
I know, but I suggest control this export at `index.ts` with `export const optimize = { LimitChunkCountPlugin }`, and at `Compiler.ts` with `optimize: { get LimitChunkCountPlugin() {...} }` instead of at here, just like `webworker.WebWorkerTemplatePlugin`